src/processor/map/map_copy_from.cpp

```diff
@@ -77,27 +78,15 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapCopyNodeFrom(
     auto& copyFrom = logicalOperator->constCast<LogicalCopyFrom>();
     const auto copyFromInfo = copyFrom.getInfo();
     const auto outFSchema = copyFrom.getSchema();
-    auto nodeTableEntry = copyFromInfo->tableEntry->ptrCast<NodeTableCatalogEntry>();
     // Map reader.
     auto prevOperator = mapOperator(copyFrom.getChild(0).get());
-    auto nodeTable = storageManager->getTable(nodeTableEntry->getTableID());
-    auto fTable =
-        FactorizedTableUtils::getSingleStringColumnFTable(clientContext->getMemoryManager());
-    const auto& pkDefinition = nodeTableEntry->getPrimaryKeyDefinition();
-    auto pkColumnID = nodeTableEntry->getColumnID(pkDefinition.getName());
-    auto sharedState = std::make_shared<NodeBatchInsertSharedState>(nodeTable, pkColumnID,
-        pkDefinition.getType().copy(), fTable, &storageManager->getWAL(),
+    auto fTable = FactorizedTableUtils::getSingleStringColumnFTable(clientContext->getMemoryManager());
+
+    // First three arguments are placeholders, check NodeBatchInsert::initGlobalStateInternal
+    auto sharedState = std::make_shared<NodeBatchInsertSharedState>(nullptr, 0, LogicalType::ANY(), 
```

Comment:
I would make change to the constructor of `NodeBatchInsertSharedState` instead.

Reply:
Fixed with acc6017393f9fe85c42fbe7ccb02d426b2b954b4